cmd/versioninfo/versioninfo.go

```diff
@@ -22,30 +22,13 @@ import (
 )
 
 var (
-	// GitBranch is the branch from which this binary was built
-	GitBranch string
-	// GitReleaseTag is the git tag from which this binary is released
-	GitReleaseTag string
-	// GitReleaseCommit is the commit corresponding to the GitReleaseTag
-	GitReleaseCommit string
-	// GitTreeState indicates if the git tree, from which this binary was built, was clean or dirty
-	GitTreeState string
-	// GitCommit is the git commit at which this binary binary was built
-	GitCommit string
-	// GitMajor is the major version of the release
-	GitMajor string
-	// GitMinor is the minor version of the release
-	GitMinor string
+	// Version is the version being released. Git commit if not a released version.
+	Version string
 )
 
 // VersionInfo returns version information for the supplied binary
 func VersionInfo(binName string) string {
 	var vi strings.Builder
```

Comment:
we can get rid of the strings builder and
`return fmt.Sprintf("%s-%s", binName, version)`